### PR TITLE
Test branch 2 nf

### PR DESCRIPTION
**Pull request by Natalie-Fischer**
Hi Neale,
I finished the first version of the phylogenetic tree visualization code. I am having some small issues with the tabs formatting, sorry. Maybe you can fix it easily. The last part with the heatmap on the side has two options: circular tree and linear tree. I am not sure which one is better or we keep both? Looking forward to your review and suggestion! Cheers, Natalie

### CODE REVIEW

File: pages/phylogenetic_trees.Rmd [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+This code chunk shows the loading of required packages:
+```{r, phylogenetic_trees_loading_packages}
+library(here)
+library(ggplot2)
+library(dplyr)
+library(ape)
+library(ggtree)
+library(treeio)
+library(ggnewscale)
+library(linelist)
```
Comment (nsbatra) [inline pages/phylogenetic_trees.Rmd:52]: Is linelist necessary? In the past there has been trouble getting it onto company/org computers. Does janitor suffice? 
Also can you convert these library() commands into one pacman::p_load()?

File: pages/phylogenetic_trees.Rmd [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+
+```
+
+First we import a phylogenetic tree in Newick file format (.nwk).
```
Comment (nsbatra) [inline pages/phylogenetic_trees.Rmd:56]: Can you explain a bit more what a newick file is, what it does/does not contain?

File: pages/phylogenetic_trees.Rmd [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+tree <- read.tree(here::here("data", "Shigella_tree.nwk"))
+
+# inspect the tree file:
+tree
+
+```
+
+Second we import a table with additional information for each sequenced sample: 
+```{r, phylogenetic_trees_loading_sampledata}
+#We read in a csv file into a dataframe format:
```
Comment (nsbatra) [inline pages/phylogenetic_trees.Rmd:69]: Can you put spaces after a #  please? For easier reading

**Review by nsbatra — APPROVED** [review]
Hi Natalie,
Thank you so much for writing this! I am SO glad to have this content for the handbook. The examples and visuals look amazing, and the organization I think works very well. 
I left a some suggestions in comments. Generally, put on the cap of someone who has never worked with genetic data but has just been asked to. Can you write more instructional normal text (outside the R chunk) covering the specialized functions and preparing the reader for the steps taken in the R chunks?
Feel free to merge now and update later, or work on it more before merging, as you wish. Thanks again - happy to discuss more.

File: pages/phylogenetic_trees.Rmd [hunk 4]
```diff
@@ -0,0 +1,345 @@
...
+<!-- ======================================================= -->
+<!-- ======================================================= -->
+<!-- ======================================================= -->
+# Visualizing phylogenetic trees {.tabset .tabset-fade}  
+
+
+<!-- ======================================================= -->
+<h2> Overview </h2> 
+## Overview {.tabset .tabset-fade .tabset-pills}
+
```
Comment (nsbatra) [inline pages/phylogenetic_trees.Rmd:33]: Can you put a visual here to draw the eye of someone skimming? See other handbook pages for examples. You could combine a couple trees at 50% size if you like - see code I give in a comment below. You can either duplicate the code up here to produce the visuals, or just snip and save them as PNGs and call them from the images folder.

**Comment by Natalie-Fischer** [comment]
Hi Neale,

Thanks for all your helpful comments! I am also happy to contribute this
part :-) I will work more on it this week to incorporate your suggestions.

Cheers, Nat

**Review by nsbatra — APPROVED** [review]
Approved! Please merge when you want to :-)

File: Shigelle_subtree_2.nwk [hunk 5]
```diff
@@ -0,0 +1 @@
+((((SRR3473848:0.001393242031,ERR025711:0.000277408259)86:0.000558293469,ERR592189:0.001671983245)14:4.286263362e-31,((((((((S19BD06884:0.001112866252,S18BD02657:0.0005557359105)100:0.001113068202,S16BD02574:0.0002781313183)97:0.0008352137802,(S18BD02761:0,S18BD02468:0,S18BD02025:0,S18BD01106:0,S18BD01024:0,S18BD00011:0,S18BD00010:0,S17BD08948:0,S17BD08179:0,S17BD07935:0,S17BD07692:0)100:0.002507055406)94:0.000277868724,S16BD04069:0.001113665699)82:0.0002786126211,S14BD03573:0.00111355797)35:1.878393283e-31,S18BD06265:0.0008348905086)82:0.0005568600631,(S16BD04164:0,S15BD08049:0)100:0.004188020113,ERR708272:0.0008348904929)79:0.0002783788048,ERR025709:0)54:0.0002785187298)100:0.001395067489,(S16BD05540:0.006751254649,ERR028677:0.0036669575)63:0.0002478908508)10:0.003066823304;
```
Comment (nsbatra) [inline Shigelle_subtree_2.nwk:1]: Can you make sure all your data files are in the data folder? Thanks! I'm not sure, but this one might not be. @Natalie-Fischer

**Review by nsbatra — APPROVED** [review]

**Comment by Natalie-Fischer** [comment]
Hi Neale,

yes you were right, it was not in the data folder. Sorry! I am pushing up
again then.

**Pull request merged**